src/Services/DataService.test.ts

```diff
@@ -797,33 +941,50 @@ describe("DataService", () => {
       fakeService.deleteRecordEpic(ActionsObservable.of(deleteRecordAction), store)
         .subscribe(noop, noop,
           () => {
-            assert.isTrue(mockAdapter.deleteItem.calledOnceWith(expectedResult.id));
+            expect(mockAdapter.deleteItem.calledOnceWith(expectedResult.id)).to.be.true;
           });
     });
 
+    it("deleteRecordEpic should call normalize before deserialize", () => {
```

Comment:
I think the test name should be written as:
```
"deleteRecordEpic should call normalize after deserialize"
```
Since the test is asserting that the normalize method was called with the expected result from the deserialize method, it is therefore testing that normalize was called after, not before. Please double-check previous tests as well